modules/cardano-node.nix

```diff
@@ -187,6 +187,7 @@ in {
         key = keyId + ".sk";
       in ''
         [ -f /run/keys/${keyId} ] && cp /run/keys/${keyId} ${stateDir}${key}
+        ulimit -c unlimited
```

Comment:
systemd.coredump.enable only sets a hook that captures cores, ulimit is still required to enable any given program to dump